activerecord/lib/active_record/associations.rb

```diff
@@ -748,9 +748,12 @@ def association_instance_set(name, association)
       # inverse detection only works on #has_many, #has_one, and
       # #belongs_to associations.
       #
-      # <tt>:foreign_key</tt> and <tt>:through</tt> options on the associations,
-      # or a custom scope, will also prevent the association's inverse
-      # from being found automatically.
+      # <tt>:foreign_key</tt> and <tt>:through</tt> options on the associations
+      # will also prevent the association's inverse from being found automatically,
+      # as will a custom scope on the inverse association. A custom scope on the
+      # association itself won't prevent automatic inverse finding as long as
+      # <tt>config.active_record.automatic_scope_inversing</tt> is set to +true+
+      # (the default for new applications).
```

Comment:
Can you put this as a separate point? It's not technically an argument on the method so including it in the args docs is confusing IMO.

Reply:
I'll simplify this and link to the [Bi-directional Associations guide](https://guides.rubyonrails.org/association_basics.html#bi-directional-associations) instead